pkg/storage/cloudimpl/external_storage.go

```diff
@@ -324,6 +324,44 @@ func URINeedsGlobExpansion(uri string) bool {
 	return containsGlob(parsedURI.Path)
 }
 
+// ExternalStorageRequiresAdminRole checks if the provided ExternalStorage URI
```

Comment:
This is a bit pedantic and bikeshed-y but I might call this `AccessesWithoutExplicitAuth` or something (can bikeshed more) since that is a purely cloud-storage determination of fact, whereas what that fact _means_ -- i.e. that it requires admin -- is what this code here is doing.